shared-core/src/kotlin/com/mountainspotter/shared/repository/MountainRepository.kt

```diff
@@ -0,0 +1,94 @@
+package com.mountainspotter.shared.repository
+
+import com.mountainspotter.shared.model.MountainPeak
+import com.mountainspotter.shared.model.Location
+
+/**
+ * Repository for mountain peak data
+ * In a real app, this would fetch from APIs like OpenStreetMap, PeakWare, or local databases
+ */
+class MountainRepository {
+    
+    // Sample data - in reality this would come from external sources
```

Comment:
```suggestion
    // todo Sample data - in reality this would come from external sources
use overpass query with gps based limit
`
[out:json][timeout:25];
// gather results
nwr["natural"="peak"]({{bbox}});
// print results
out geom; `
```